src/test/ui/rfc-2632-const-trait-impl/call-generic-method-nonconst.stderr

```diff
@@ -1,26 +1,17 @@
-error[E0277]: can't compare `S` with `S` in const contexts
-  --> $DIR/call-generic-method-nonconst.rs:18:34
+error[E0277]: the trait bound `S: ~const Foo<const>` is not satisfied
```

Comment:
This might be confusing. perhaps we should only display one of the `~const` and `const`? Could be followup though.

Reply:
yea, these are completely bogus generic params and we should only display them as generic params with `-Zverbose` and otherwise use the `~const Trait` syntax